koji_containerbuild/plugins/cli_containerbuild.py

```diff
@@ -56,9 +56,15 @@ def print_task_result(task_id, result, weburl):
     except TypeError:
         pass
 
+    user_warnings = result.pop('user_warnings', None)
```

Comment:
I think we should avoid such special handling for user warnings

Reply:
Dropped